devnet/bob.Dockerfile

```diff
@@ -0,0 +1,35 @@
+# Copyright 2021 ChainSafe Systems (ON)
+# SPDX-License-Identifier: LGPL-3.0-only
+
+FROM golang:1.17
+
+ARG DD_API_KEY=somekey
+RUN DD_API_KEY=${DD_API_KEY} DD_AGENT_MAJOR_VERSION=7 DD_INSTALL_ONLY=true DD_SITE="datadoghq.com" bash -c "$(curl -L https://s3.amazonaws.com/dd-agent/scripts/install_script.sh)"
+
+WORKDIR /gossamer
+
+COPY go.mod go.sum ./
+RUN go mod download
+
+COPY . .
+
+RUN go install -trimpath github.com/ChainSafe/gossamer/cmd/gossamer
+
+# use modified genesis-spec.json with only 3 authority nodes
+RUN cp -f devnet/chain/gssmr/genesis-spec.json chain/gssmr/genesis-spec.json
+
+ARG key
+RUN test -n "$key"
+ARG pubip
+RUN test -n "$pubip"
+
+ENV key=${key}
+ENV pubip=${pubip}
+
+RUN gossamer --key=${key} init
+
+RUN go run devnet/cmd/update-dd-agent-confd/main.go -n=gossamer.local.devnet -t=key:${key} > /etc/datadog-agent/conf.d/openmetrics.d/conf.yaml
+
+ENTRYPOINT service datadog-agent start && gossamer --key=${key} --bootnodes=/ip4/10.5.0.2/tcp/7001/p2p/12D3KooWMER5iow67nScpWeVqEiRRx59PJ3xMMAYPTACYPRQbbWU --publish-metrics --rpc --pubip=${pubip}
```

Comment:
I don't see a case where we do want to adjust the bootnodes for the `gssmr` devnet.

Reply:
At least the IP address `10.5.0.2` could be changed if we change the docker network right?

Reply:
In the next PR I do actually have to change the bootnode ports, so port will be configurable on `bob.Dockerfile`.  I'm trying to hardcode the configuration for `alice.Dockerfile` since that is the lead node.  I think being explicit in this case  shows that everything is dependant on alice.